2020-03-20 23:44:50,104 - instabot version: 0.115.0 (bot) - INFO - Instabot version: 0.115.0 Started
2020-03-20 23:44:50,104 - instabot version: 0.115.0 (bot) - DEBUG - Bot imported from /usr/local/lib/python2.7/dist-packages/instabot/bot/bot.pyc
2020-03-20 23:44:50,105 - instabot version: 0.115.0 (api_login) - DEBUG - Loading cookies
2020-03-20 23:44:50,106 - instabot version: 0.115.0 (api_login) - DEBUG - Loading uuids
2020-03-20 23:44:50,106 - instabot version: 0.115.0 (api_login) - INFO - Recovery from /var/www/html/example.com/public_html/cron/insta/config/menabungtanah_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=a30092fa-e99d-46c6-b2c6-a80dfd9fe9e9
- uuid=8aacbd94-8261-42c3-ae0d-d5476e870db9
- client_session_id=dd54da24-561a-4af9-a615-72915b144618
- device_id=android-d13a49bfddbe7ce7
2020-03-20 23:44:50,106 - instabot version: 0.115.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-03-20 23:44:50,496 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [400]>
2020-03-20 23:44:50,496 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/feed/timeline/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:50,497 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:50,498 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:50,499 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:50,848 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [400]>
2020-03-20 23:44:50,849 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/feed/reels_tray/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:50,849 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:50,850 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:50,850 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:51,194 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: direct_v2/ranked_recipients/ returned response: <Response [400]>
2020-03-20 23:44:51,194 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/direct_v2/ranked_recipients/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:51,195 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:51,195 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:51,195 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:51,541 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: direct_v2/ranked_recipients/ returned response: <Response [400]>
2020-03-20 23:44:51,542 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/direct_v2/ranked_recipients/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:51,542 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:51,546 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:51,546 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:51,894 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: direct_v2/inbox/ returned response: <Response [400]>
2020-03-20 23:44:51,895 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/direct_v2/inbox/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:51,896 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:51,896 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:51,896 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:52,208 - instabot version: 0.115.0 (api) - DEBUG - GET to endpoint: direct_v2/get_presence/ returned response: <Response [400]>
2020-03-20 23:44:52,208 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/direct_v2/get_presence/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:52,208 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:52,211 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:52,211 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:52,562 - instabot version: 0.115.0 (api) - DEBUG - GET to endpoint: news/inbox/?limited_activity=true&show_su=true returned response: <Response [400]>
2020-03-20 23:44:52,563 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/news/inbox/%253Flimited_activity%253Dtrue%2526show_su%253Dtrue", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:52,563 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:52,563 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:52,564 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:52,917 - instabot version: 0.115.0 (api) - DEBUG - GET to endpoint: users/profile_notice/ returned response: <Response [400]>
2020-03-20 23:44:52,917 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/users/profile_notice/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:52,918 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:52,918 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:52,918 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:53,279 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: discover/explore/ returned response: <Response [400]>
2020-03-20 23:44:53,279 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/discover/explore/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:53,280 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:53,280 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:53,281 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:53,284 - instabot version: 0.115.0 (api) - INFO - Login flow failed, the cookie is broken. Relogin again.
2020-03-20 23:44:53,285 - instabot version: 0.115.0 (api_login) - DEBUG - Loading cookies
2020-03-20 23:44:53,285 - instabot version: 0.115.0 (api_login) - DEBUG - Loading uuids
2020-03-20 23:44:53,287 - instabot version: 0.115.0 (api_login) - INFO - Recovery from /var/www/html/example.com/public_html/cron/insta/config/menabungtanah_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=a30092fa-e99d-46c6-b2c6-a80dfd9fe9e9
- uuid=8aacbd94-8261-42c3-ae0d-d5476e870db9
- client_session_id=2749fddf-2f8d-45ed-aa85-0d6f0de39045
- device_id=android-d13a49bfddbe7ce7
2020-03-20 23:44:53,287 - instabot version: 0.115.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2020-03-20 23:44:53,660 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [400]>
2020-03-20 23:44:53,661 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/accounts/get_prefill_candidates/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:53,661 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:53,661 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:53,663 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:54,029 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [400]>
2020-03-20 23:44:54,029 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/qe/sync/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:54,029 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:54,030 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:54,030 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:54,370 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [400]>
2020-03-20 23:44:54,370 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/launcher/sync/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:54,370 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:54,370 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:54,371 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:54,723 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [400]>
2020-03-20 23:44:54,725 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/accounts/contact_point_prefill/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:54,725 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:54,725 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:54,726 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:55,079 - instabot version: 0.115.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2020-03-20 23:44:55,080 - instabot version: 0.115.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "challenge_required", "challenge": {"url": "https://i.instagram.com/challenge/?next=/api/v1/accounts/login/", "api_path": "/challenge/", "hide_webview_header": true, "lock": true, "logout": false, "native_flow": true}, "status": "fail"}
2020-03-20 23:44:55,080 - instabot version: 0.115.0 (api) - ERROR - Request returns 400 error!
2020-03-20 23:44:55,080 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: challenge_required
2020-03-20 23:44:55,081 - instabot version: 0.115.0 (api) - INFO - Instagram's error message: {}
2020-03-20 23:44:55,081 - instabot version: 0.115.0 (api) - ERROR - Failed to login go to instagram and change your password
2020-03-20 23:44:55,081 - instabot version: 0.115.0 (api) - INFO - Username or password is incorrect.
2020-03-20 23:48:31,755 - instabot version: 0.115.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: /var/www/html/example.com/public_html/cron/insta/config/nurchuliss.checkpoint
2020-03-20 23:48:31,757 - instabot version: 0.115.0 (bot) - INFO - Total requests: 120
